docs/xostor/xostor.md

```diff
@@ -867,13 +867,12 @@ This feature is currently experimental and not covered by [Vates Pro Support](ht
 
 On each host, create a new PV and VG using your cache devices:
 ```
-pvcreate linstor_group_cache <CACHE_DEVICES>
 vgcreate linstor_group_cache <CACHE_DEVICES>
 ```
 
 Then you can enable the cache with a few commands using the linstor controller.
 
-Verify the group to modify, it must start with "xcp-sr-":
+Verify the group to modify, it must start with "xcp-sr-" (generally `xcp-sr-linstor_group`):
```

Comment:
Since you use the thin for all other example below, maybe thin would be better here?